include/flux/core/inline_sequence_base.hpp

```diff
@@ -408,6 +408,9 @@ struct inline_sequence_base {
                  std::indirectly_writable<Iter, element_t<Derived>>
     constexpr auto output_to(Iter iter) -> Iter;
 
+    template <sequence Seq, typename Cmp = std::ranges::less>
+    constexpr auto set_union(Seq&& seq, Cmp cmp = {});
```

Comment:
I was thinking about if I should add "inline" version of the call. Maybe in case of `set_difference` of `set_intersection` it makes more sense, but I agree with you and removing "inline" call.

Reply:
This declaration still needs to be removed